pkg/analyze/analyze.go

```diff
@@ -185,10 +187,19 @@ func (a *Analyzer) handleLine(line []byte) error {
 		}
 		ipStats.LastSize = ipStats.Size
 	}
-	a.ipInfo[clientPrefix] = ipStats
+	if a.serverIpInfo[logItem.Server] == nil {
+		a.serverIpInfo[logItem.Server] = make(map[netip.Prefix]IPStats)
+	}
+	a.serverIpInfo[logItem.Server][clientPrefix] = ipStats
 	return nil
 }
-func (a *Analyzer) PrintTopValues(displayRecord map[netip.Prefix]time.Time, sortBy string) {
+
+type KeyIndex struct {
```

Comment:
What about just using `KeyIndex` as the map key, so that we don't have to handle nested maps?